sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -801,15 +815,20 @@ static int sof_card_dai_links_create(struct device *dev,
 		return -EINVAL;
 
 	for (; adr_link->num_adr; adr_link++) {
-		val = create_sdw_codec_dai(dev, links, id, cpus, adr_link);
-		if (val < 0) {
+		ret = create_sdw_codec_dai(dev, links, &id, cpus, adr_link);
+		if (ret < 0) {
 			dev_err(dev, "failed to create dai link %d", id);
 			return -ENOMEM;
 		}
-
-		id += val;
 	}
 
+	/*
+	 * dai link id of SSP & DMIC & HDMI are based on last
+	 * link id used by sdw link. The dai_id is not always equal
+	 * to id for dai link id may be changed by some sdw codecs.
```

Comment:
Let's capitalise: "DAI," "ID," not sure what the preferred way for "sdw" is - it isn't an acronym, so, maybe in small is ok.

Reply:
In your comment you say, that you unified the acronym style by writing them all in lower-case, but here it's still mixed. And I don't think "hdmi" is a widely-accepted style, so I'd really use "HDMI," "DAI," "ID"... But let's see what others say, maybe I'm wrong on this.

Reply:
yes, I grep git log and sound,   HDMI DAI ID is more popular

Reply:
...and still this comment hasn't been updated.

Reply:
sorry, missed, done